src/core/blockchain/zilliqa/stats.ts

```diff
@@ -26,9 +26,11 @@ export class Stats extends GenericStats<Client> {
         );
 
         let availableToDelegate = new BigNumber(0);
-        if (new BigNumber(data.balance.available).gt(config.amountToKeepInAccount)) {
```

Comment:
we need to be sure account.type is always defined, otherwise this will fail

Reply:
We have this state migration in place

![image](https://user-images.githubusercontent.com/32679726/96281645-4899da00-0fe2-11eb-8c3f-b3a2affd62e2.png)